src/main/resources/db/migration/V1.0.0__create_songs_table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS songs(
+ id INTEGER NOT NULL GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+ artistName VARCHAR(255) NOT NULL,
+ trackName VARCHAR(255) NOT NULL,
+ trackTime TIME NOT NULL,
+ genre VARCHAR (255) NOT NULL,
+ decade INTEGER NOT NULL,
```

Comment:
A czemu nie rok 🙂 ? Mając lata bez problemu pogrupujemy utwory wg dekad, w drugą stronę to się nie uda.